packages/esm-patient-medications-app/src/add-drug-order/drug-order-form.component.tsx

```diff
@@ -238,6 +242,22 @@ export function DrugOrderForm({ initialOrderBasketItem, onSave, onCancel, prompt
 
   const medicationOrderFormSchema = useCreateMedicationOrderFormSchema();
 
+  const { currentProvider: _provider, user } = useSession();
+  const currentProvider: Provider = useMemo(
+    () => ({
+      ..._provider,
+      person: user.person,
+    }),
+    [_provider, user],
+  );
+  const { data: providers } = useProviders(config.prescriberProviderRoles);
+  const providersWithCurrentUser = useMemo(() => {
+    if (providers == null) {
+      return providers;
+    }
+    return providers.some((p) => p.uuid === currentProvider.uuid) ? providers : [currentProvider, ...providers];
+  }, [providers, currentProvider]);
```

Comment:
Is this not suppose to include the current provider if the provider list is null?  It looks like  line 258 adds the current provider if it's not in the list, but yet if there are no providers it just returns null?  (Not saying this is necessarily wrong, just trying to figure out what is happening here).